apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -860,13 +860,28 @@ get_account_transactions(Account, Req) ->
                 end,
             Res =
               lists:sort(
-                  fun({HeaderA, _}, {HeaderB, _}) ->
-                      aec_headers:height(HeaderA) > aec_headers:height(HeaderB)
+                  fun({HeaderA, SignedTxA}, {HeaderB, SignedTxB}) ->
+                      HeightA = aec_headers:height(HeaderA),
+                      HeightB = aec_headers:height(HeaderB),
+                      case HeightA =:= HeightB of
+                          false ->
+                              HeightA > HeightB;
+                          true ->
+                              TxA = aec_tx_sign:data(SignedTxA),
+                              TxB = aec_tx_sign:data(SignedTxB),
+                              case aec_tx:origin(TxA) =:= aec_tx:origin(TxB) of
+                                  true ->
+                                      aec_tx:nonce(TxA) > aec_tx:nonce(TxB);
+                                  false ->
+                                      aec_tx:origin(TxA) > aec_tx:origin(TxB)
+                              end
+                      end
```

Comment:
Why not
```erlang
fun({HeaderA, SignedTxA}, {HeaderB, SignedTxB}) ->
      HeightA = aec_headers:height(HeaderA),
      HeightB = aec_headers:height(HeaderB),
      TxA = aec_tx_sign:data(SignedTxA),
      TxB = aec_tx_sign:data(SignedTxB),
      {HeightA, aec_tx:origin(TxA), aec_tx:nonce(TxA)} >
      {HeightB, aec_tx:origin(TxB), aec_tx:nonce(TxB)}
  end
```
?
    
    {aec_headers:height(HeaderA), aec_tx_sign:data(SignedTxA), aec_tx:
end
```

Reply:
yes, this is cleaner. Thanks